client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -114,25 +115,38 @@ export const RequestLineEditForm = ({
   const formatNumber = useFormatNumber();
   const { lines } = useRequest.line.list();
   const requestedQuantity = draftLine?.requestedQuantity ?? 0;
+  const originalItemName = lines?.find(({ item }) => item.id === item?.id)
+    ?.itemName;
 
   return (
     <RequestLineEditFormLayout
       Top={
-        <StockItemSearchInputWithStats
-          autoFocus={!item}
-          openOnFocus={!item}
-          width={600}
-          disabled={disabled}
-          currentItemId={item?.id}
-          onChange={(newItem: ItemRowWithStatsFragment | null) =>
-            newItem && onChangeItem(newItem)
-          }
-          extraFilter={
-            disabled
-              ? undefined
-              : itemRow => !lines?.some(({ item }) => itemRow.id === item.id)
-          }
-        />
+        <>
+          {(disabled && (
+            <BasicTextInput
+              value={`${item?.code}     ${originalItemName}`}
```

Comment:
Can't the item.code change in a merge?

Reply:
Yeah, but I was told only the item name should stay the same

Reply:
Ah yes, thanks remember now